contracts/FanDonation.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: agpl-3.0
+
+pragma solidity ^0.6.0;
```

Comment:
Not sure why solhint is still complaining since clearly 0.6.0 > 0.5.8, but feel free to add the `compiler-version` rule to turn this warning off to `.solhint.json`:

```
{
  "extends": "solhint:recommended",
  "rules": {
    "compiler-version": "off"
  }
}
```

Reply:
Oh wait, it's expecting a version 0.5.x where x > 8 based on the `^` semver syntax. Definitely ignore it then with the solhint change.